packages/core-components/src/layout/Header/Header.tsx

```diff
@@ -26,6 +26,8 @@ const useStyles = makeStyles<BackstageTheme>(theme => ({
   header: {
     gridArea: 'pageHeader',
     padding: theme.spacing(3),
+    height: 'fit-content',
+    // Where does this number come from? :/
```

Comment:
Probably just that some pages have very little content - or even no content at all while waiting for the page to load and give what the title should be. In those cases it looks bad with a miniature header that then jumps in size.

However... Those cases could probably be handled a lot better

Reply:
The lowest hanging fruit for improvement in my perspective here would be to assign this number to a `const` that indicates its intent and then using the `const` here for clarity.